<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>433</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Vivaldi</family>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:#FAEEDD;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vladimir Script</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #79553D;</string>
   </property>
   <property name="text">
    <string>The Coffee guide</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #EEE8AA;</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Выберете вид кофе</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #79553D;</string>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Цена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="z_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Молотый/в зернах:</string>
   </property>
  </widget>
  <widget class="QLabel" name="so_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Степень обжарки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Название:</string>
   </property>
  </widget>
  <widget class="QLabel" name="o_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #BDECB6;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Объем упаковки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #79553D;</string>
   </property>
   <property name="text">
    <string>Хар-ки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #79553D;</string>
   </property>
   <property name="text">
    <string>&lt;-Выберете вид кофе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(213, 233, 255);
border-radius:11px;}

QPushButton:hover{background-color: rgb(213, 233, 200);color: #ffffff}</string>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(213, 233, 255);
border-radius:11px;}

QPushButton:hover{background-color: rgb(213, 233, 200);color: #ffffff}
</string>
   </property>
   <property name="text">
    <string>Добавить новый вид</string>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>price_label</zorder>
  <zorder>z_label</zorder>
  <zorder>so_label</zorder>
  <zorder>id_label</zorder>
  <zorder>name_label</zorder>
  <zorder>o_label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
